nyxx_commander/lib/src/CommandHandler.dart

```diff
@@ -19,7 +19,7 @@ abstract class CommandEntity {
   CommandEntity? get parent;
 
   /// A list of valid command names
-  List<String> get commandNames => [if (this.name.isNotEmpty) this.name, ...this.aliases];
+  List<String> get commandNames => [if (this.name.isNotEmpty) this.name.toLowerCase(),]+aliases.map((e) => e.toLowerCase()).toList();
```

Comment:
`...aliases.map((e) => e.toLowerCase()).toList()` would be better, but i'm not sure if that's possible

Reply:
Updated.